### PR TITLE
Remove jQueryUI, interest and expertise fields - Profile cleanup

### LINKED ISSUE #6320: Profile - Remove jQuery UI
**Summary**

Remove `Interests (tags)` field and jQuery UI.

**Rationale**

Currently the edit profile page currently still depends on jQuery UI. The release we use, v1.10.3 is no longer supported by the jQuery dev team and has a known vulnerability(https://snyk.io/test/npm/jquery-ui/1.10.4)

The only reason we still have jQueryUI on this page is for the `Interests (tags)` field.

**Audience**

All signed-in users that visit the edit profile page

**Acceptance Criteria**

- [ ] The `Interests (tags)` field is removed fro the UI
- [ ] jQueryUI is removed as a dependency
- [ ] All other functionality still works as before
- [ ] View profile page works as before

### PR DESCRIPTION
**Pull request by schalkneethling**
Turns out this allowed me to clean up quite a bit of code :) @escattone I am pinging you for review as you probably know the most history around this. Let me know if I missed anything.

Fix #6320

### CODE REVIEW

**Comment by peterbe** [comment]
@schalkneethling Lemme know if there's anything I can do to help to conflict resolve this. 
A tip is that when you confront each files conflict, you can press to resolve the "incoming change" almost always and when you're done, you run `black` on all the files you resolved.

**Comment by schalkneethling** [comment]
@peterbe @escattone I ran:

```
docker-compose exec web black kuma
```

but I get the following error:

```
OCI runtime exec failed: exec failed: container_linux.go:346: starting container process caused "exec: \"black\": executable file not found in $PATH": unknown
```

**Comment by schalkneethling** [comment]
> @peterbe @escattone I ran:
> 
> ```
> docker-compose exec web black kuma
> ```
> 
> but I get the following error:
> 
> ```
> OCI runtime exec failed: exec failed: container_linux.go:346: starting container process caused "exec: \"black\": executable file not found in $PATH": unknown
> ```

After fighting with `brew` for some time I finally got Python 3.8 installed. I then did `poetry use env 3` which created a new virtualenv `Creating virtualenv kuma-UBQC6x8Y-py3.8`

Now when I run `poetry run black kuma` from the root of Kuma, I get:

```
[OSError]
[Errno 2] No such file or directory
```

😿

**Comment by peterbe** [comment]
> > @peterbe @escattone I ran:
> > ```
> > docker-compose exec web black kuma
> > ```
> > 
> > 
> > but I get the following error:
> > ```
> > OCI runtime exec failed: exec failed: container_linux.go:346: starting container process caused "exec: \"black\": executable file not found in $PATH": unknown
> > ```
> 
> After fighting with `brew` for some time I finally got Python 3.8 installed. I then did `poetry use env 3` which created a new virtualenv `Creating virtualenv kuma-UBQC6x8Y-py3.8`
> 
> Now when I run `poetry run black kuma` from the root of Kuma, I get:
> 
> ```
> [OSError]
> [Errno 2] No such file or directory
> ```
> 
> 😿

Whenever you get that, run:

```
poetry shell
python ....
```
then it won't try to hide the actual error.

**Comment by peterbe** [comment]
@schalkneethling By the way, you need to run `docker-compose pull && make build-base VERSION=latest && dc up` so that `black` gets installed inside the container.

**Comment by schalkneethling** [comment]
> @schalkneethling By the way, you need to run `docker-compose pull && make build-base VERSION=latest && dc up` so that `black` gets installed inside the container.

The one thing I did not do :) Well, I did do `docker-compose down` and then `docker-compose up -d`
I am pretty sure in the past it would pull the latest if there was anything. Anyhow, pulling :)

**Comment by schalkneethling** [comment]
> **LATER NOTE:**
> @schalkneethling I just realized that I may have misunderstood "remove jquery-ui" as removing it completely from all of kuma, where you may have meant it as just being removed from the profile page? Even if that's the case, there is still something off with the `PIPELINE_JS` setting in `kuma/settings/common.py`

Indeed, this is focused on edit and view profile/account/settings :) Apologies for the confusion.

File: kuma/settings/common.py [hunk 1]
```diff
@@ -886,19 +886,14 @@ def pipeline_one_scss(slug, **kwargs):
...
+        },
     },
-    "banners": {
-        "source_filenames": (
-            "js/components/banners/utils/banners-event-util.js",
-            "js/components/banners/utils/banners-state-util.js",
-            "js/components/banners/banners.js",
+    'users': {
+        'source_filenames': (
+            'js/users.js',
```
[the remark this answers is not in the document]
Comment (schalkneethling) [reply kuma/settings/common.py]: Oh boy! That is probably due to some merge conflict resolutions with Black and friends. Let me look over that again.

[the remark this answers is not in the document]
Comment (schalkneethling) [reply kuma/settings/common.py]: This has been resolved. Thanks Ryan

**Comment by schalkneethling** [comment]
I am running everything in `ci-python` to try and see where Travis is falling over. I will update the PR once I have found the problem.

**Comment by schalkneethling** [comment]
@escattone Ok, I have persisted and now all the things are passing. 🎉

File: kuma/settings/common.py [hunk 2]
```diff
@@ -886,8 +886,8 @@ def pipeline_one_scss(slug, **kwargs):
             # placed in the kuma/javascript/dist/ directory.
             "react.js",
         ),
-        "output_filename": "build/js/react-main.js",
-        "extra_context": {"defer": True},
+        "output_filename": "build/js/banners.js",
```
Comment (peterbe) [inline kuma/settings/common.py]: What?!

Comment (schalkneethling) [reply kuma/settings/common.py]: Ugh yeah, somehow the merge and conflicts just played havoc with this file. Let me sort this one out as well.

Comment (schalkneethling) [reply kuma/settings/common.py]: Sorted.

**Comment by escattone** [comment]
I'm going to go ahead and merge since @peterbe's requested change has also been satisfied.

**Pull request merged**